### PR TITLE
Insecticide resistance results and discussion

### PR DESCRIPTION
**Pull request by cclarkson**

### CODE REVIEW

File: main.tex [hunk 1]
```diff
@@ -189,7 +189,7 @@ \subsection*{Population structure}
 \end{figure}
 
 %
-Of particular importance to the Phase 2 data release are the three \textit{An. coluzzii} populations, Cote d'Ivoire (CIcol), Guinea (GNcol) and Ghana (GHcol) (Figure \ref{sample_map}). 
+Of particular importance to the Phase 2 data release are the three \textit{An. coluzzii} populations, C\^{o}te d'Ivoire (CIcol), Guinea (GNcol) and Ghana (GHcol) (Figure \ref{sample_map}). 
 %
 In Phase 1, just two of the nine populations belonged to this species and these, Burkina Faso (BFcol) and Angola (AOcol), appeared quite different to each other in population structure (principal component analysis (PCA), doubleton sharing and F\raisebox{-.4ex}{\scriptsize ST}).
```
Comment (hardingnj) [inline main.tex]: _suggestions_: 
- two of nine populations were _coluzzii_
- I don't think "appeared" is good, maybe be more definite

File: main.tex [hunk 2]
```diff
@@ -216,27 +216,19 @@ \subsection*{Population structure}
 %
 The Phase 2 Gambian (GM) population is strikingly similar in multiple aspects to the the samples from neighbouring Guinea Bissau (GW).
 %
-Both populations have highly 'hybridised' autosomes (Figure \ref{aim}), fall together in early principal components (Figure \ref{pca}), show high doubleton sharing with each other but not between other populations (Figure \ref{doubletons}) and are weakly differentiated (F\raisebox{-.4ex}{\scriptsize ST} = 0.007 Figure \ref{fst}).
+Both populations have highly 'hybridised' autosomes (Figure \ref{aim}), fall together in early principal components (Figure \ref{pca}), show high doubleton sharing with each other but not between other populations (Figure \ref{fstdbl}b) and are weakly differentiated (F\raisebox{-.4ex}{\scriptsize ST} = 0.007 Figure \ref{fstdbl}a).
```
Comment (hardingnj) [inline main.tex]: I think we can steer clear of "hybridised" here, I think we can say similar patterns of species painting or something. I don't think they are hybrid populations, likely that the M/S alleles just break down in this region (?).

File: main.tex [hunk 3]
```diff
@@ -216,27 +216,19 @@ \subsection*{Population structure}
...
 %
 GQgam clusters with Ugandan (UGcol), Ghanaian (GHcol) and Cameroonian (CMcol) \textit{An. gambiae} in early PCA components (Figure \ref{pca}), though has elevated levels of genetic differentiation when compared to other 
-\textit{An. gambiae vs. An. gambiae} F\raisebox{-.4ex}{\scriptsize ST} values (exluding FRgam) (Figure \ref{fst}). 
+\textit{An. gambiae vs. An. gambiae} F\raisebox{-.4ex}{\scriptsize ST} values (exluding FRgam) (Figure \ref{fstdbl}a). 
 %
-FRgam, however, is an outlier in principal components three and four, has almost no doubleton sharing with any other populations and the only population more genetically differentiated in pairwise comparisons is Kenya (KE) which known to be highly bottlenecked \cite{Ag1000gConsortium2017} (Figure \ref{fst}).
+FRgam, however, is an outlier in principal components three and four, has almost no doubleton sharing with any other populations and the only population more genetically differentiated in pairwise comparisons is Kenya (KE) which known to be highly bottlenecked \cite{Ag1000gConsortium2017} (Figure \ref{fstdbl}).
```
Comment (hardingnj) [inline main.tex]: I think we can drop "however".
There is a missing "is" in "which (is) known"

File: main.tex [hunk 4]
```diff
@@ -260,32 +252,59 @@ \subsection*{Insecticide resistance}
...
-We then combined these data with data on nucleotide variation within the \textit{Vgsc} gene, which is the target for pyrethroids, to provide an estimate for the population frequencies of mosquitoes carrying some form of \textit{kdr} target-site pyrethroid resistance, amplifications at either of the two P450 loci, or both (Figure 13). 
 %%
 
+
+%%
+We combined the amplification data with data on nucleotide variation within the \textit{Vgsc} gene (Clarkson et al, 2019), the target for pyrethroids, to provide an estimate for the population frequencies of mosquitoes carrying some form of \textit{kdr} target-site pyrethroid resistance, amplifications at either of the two P450 loci, or both (Figure \ref{ir}). 
+%
+PBO impregnated nets may only offer benefits over traditional LLINs where metabolic resistance to pyrethroids is present at appreciable frequencies.
+%
+Fortunately our analysis reveals several features of the pyrethroid resistance landscape that could determine the effectiveness of PBO bed-nets and therefore inform national malaria control programmes (NMCPs) with respect to bed-net roll-out.
```
Comment (hardingnj) [inline main.tex]: minus Fortunately?

File: main.tex [hunk 5]
```diff
@@ -320,11 +339,56 @@ \subsection*{Direct recombination map}
...
+%%
+
+%%
+Pyrethroid insecticides are currently used in all LLINs, consequently the pervasive increase in pyrethroid resistance detected across Sub-Saharan Africa is raising fears of vector control failure ultimately leading to increases in malaria morbidity and morbidity \cite{churcher2016, Hemingway2016}.
+%
+To ameliorate the risk of vector control failure, pyrethoid LLINs containing the synergist PBO have been in development and testing for some time \cite{darriet2011, protopopoff2018}.
+%
+PBO enhances the potency of certain insecticides (including pyrethroids), where resistance to these chemicals is cytochrome P450 mediated (metabolic resistance).
+%
+A study using  meta-analysis to parameterise mathematical models suggests that PBO bednets could avert up to 0.5 clinical cases per person per year in some locations \cite{churcher2016}.
```
Comment (hardingnj) [inline main.tex]: Is this definitely right?? Per case per year perhaps?

File: main.tex [hunk 6]
```diff
@@ -320,11 +339,56 @@ \subsection*{Direct recombination map}
...
+%
+\textbf{2.} Regions with both high target-site and high metabolic resistance associated loci frequencies, many individuals in these regions carry markers for both modes of resistance.
+%
+The outcome of PBO nets is less intuitive in these regions, as the addition of the synergist can only reduce the metabolic resistance mediated fraction of the pyrethroid resistance phenotype.
+%
+\textbf{3.} Regions with low frequency or no markers for metabolic resistance, here the combination of low potential benefits and the increased cost of the nets may render roll-out inadvisable.
+%%
+
+%%
+It must be reiterated that our results are a snapshot of pyrethroid resistance at the time of sample collection (Table \ref{table:sampling_locations}, and use four resistance associated loci which may not explain all phenotypic variation in all locations \cite{donnelly2009}.
```
Comment (hardingnj) [inline main.tex]: I think we also need a sentence making it crystal clear that there may be other non amplification mediated cyp6 resistance

File: main.tex (hunk 4 above)
Comment (EricRLucas) [reply main.tex]: agreed

File: main.tex [hunk 7]
```diff
@@ -260,32 +252,59 @@ \subsection*{Insecticide resistance}
...
+%
+Further east, in samples from Gabon, Cameron, Angola and Ghana, lower frequencies of P450 amplification (Gabon 1\%, Cameroon 7\%, Angola 15\%, Ghana 27\%), alongside high \textit{kdr} frequencies, suggest the net gain of PBO inclusion may be small.
+%
+Eastern populations appear split by the rift valley system, with Uganda presenting high frequencies of P450 mutations (81\%), and therefore the potential for PBO gains, whereas Kenya has almost no P450 amplification (4\%) (Figure \ref{ir}).
+%
+On the two island population of Bioko and Mayotte no pyrethroid resistance, target-site or metabolic, markers are detected.
+%
+In countries where sampling covers sympatric \textit{An. gambiae} and \textit{An. coluzzii}, Burkina Faso and Ghana, we find similar profiles of resistance loci across species, reinforcing recent findings of highly porous species boundaries where insecticide resistance is concerned (\cite{Ag1000gConsortium2017}, Clarkson et al 2019, Lucas et al, 2019).
```
Comment (EricRLucas) [inline main.tex]: At least on the level of CNVs, I don't agree with this statement. There is very little CNV overlap between the two species, even when they come from the same site.

File: main.tex (hunk 6 above)
Comment (EricRLucas) [reply main.tex]: Also, when we first introduce the topic of CNVs, we say that their importance in pyrethroid resistance is an assumption, but after that we seem to be drifting more and more into stating this association as a given (eg: talking about them as resistance loci, or a resistance topology. I think we need a reminder that we don't actually know yet whether these CNVs are linked to pyrethroid resistance.

File: main.tex (hunk 2 above)
Comment (cclarkson) [reply main.tex]: Agree, I don't think they are hybridised in any simple sense either. I have altered the text.

File: main.tex (hunk 3 above)
Comment (cclarkson) [reply main.tex]: fixed.

File: main.tex (hunk 4 above)
Comment (cclarkson) [reply main.tex]: fixed.

File: main.tex (hunk 5 above)
Comment (cclarkson) [reply main.tex]: This is correct, each person has a number of clinical cases a year, on average, per person, this is reduced by 0.5 clinical cases.

File: main.tex (hunk 7 above)
Comment (cclarkson) [reply main.tex]: Agreed, doesn't make sense, removed the sentence.

File: main.tex (hunk 6 above)
Comment (cclarkson) [reply main.tex]: Agreed, I have added the following sentence and have made a note to temper this language throughout the manuscript.

It is likely that the four loci we analysed do not explain all phenotypic variation in all locations \cite{donnelly2009}, and although all loci involve resistance associated genomic regions, the specific P450 CNVs used here have yet to undergo molecular validation (lucas et al 2019).

**Comment by cclarkson** [comment]
Thanks for all the comments @EricRLucas,  @hardingnj

File: main.tex (hunk 5 above)
Comment (hardingnj) [reply main.tex]: Which implies that the average person has a clinical case at the absolute least once every 2 years! Cool- thanks. Fig 5A seems to be where they get that number from btw.

**Pull request merged**